### DISCUSSION TITLE
BentoML 0.8.4 release!

### DISCUSSION THREAD

**Discussion by parano** [bentoml/BentoML#971] [General]
BentoML 0.8.4 has just arrived with tons of new features and improvements!

https://github.com/bentoml/BentoML/releases/tag/v0.8.4

Thank you all for your contributions to this release! @bojiang @joshuacwnewton @guy4261 @Sharathmk99 @co42 @jackyzha0 @Korusuke @akainth015 @omrihar @yubozhao

A few quick questions for the community:

* What is your favorite feature in 0.8.4?
* Any questions about the new features or the breaking changes in this release?
* What are the features in the [issues list](https://github.com/bentoml/BentoML/issues?q=is%3Aissue+is%3Aopen+label%3A%22new+feature%22) that you want the most and why?

**Comment by omrihar** [comment]
Good job! I especially like the separation of training and bentoml service creation :)

I'm really excited about the future of BentoML!

**Comment by yubozhao** [comment]
Love the Ngrok integration, super easy to demo in a meetup now.

**Reply by parano** [reply]
Also when working with Google Colab or a remote Jupyter Notebook environment, this makes it possible to try out the API server easily!

**Comment by withsmilo** [comment]
Great job!
I have been waiting a long time for `bentoml containerize command` to be released!

**Reply by parano** [reply]
We have a few more model management related features coming in next release! The goal is to make it easy to push locally saved bundles to a remote YataiService, pulling down saved bundles from remote YataiService to local Yatai, and allow the user to `prune` locally saved bundles to free up space on the dev machine.

This should make collaborating and sharing models more easily in a team setting, and simplify building model training pipelines and CI/CD pipelines with BentoML.

**Comment by HenryDashwood** [comment]
You guys are on fire right now!
